apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/SimpleManagedTracer.cs

```diff
@@ -17,7 +17,13 @@
 using Google.Protobuf.WellKnownTypes;
 using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Diagnostics;
+#if !NETSTANDARD1_5
+// TODO: Remove when we support .NET 4.6.1
```

Comment:
Acknowledged. I removed the TODOs and changed these to look for NET45 instead so a target of .NET 4.6.1 will automatically pickup the AsyncLocal code.